Explorer/Assets/Scripts/ECS/Unity/Systems/SpecialComponentsID.cs

```diff
@@ -0,0 +1,16 @@
+namespace Unity.ECS.Components
+{
+    public static class SpecialEntityId
```

Comment:
Please rename the file to match the class name, move it to the `Bridge` directory, and unify the IDs with ones in `EntityFactory`